pkgs/tools/networking/socat/default.nix

```diff
@@ -16,6 +16,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-ZpCp+ZkEV7UFCXonK78sv0zDVXYXb3ZkbjUksOkcF2M=";
   };
 
+  patches = [
+    # This adds missing feature checks for TCP_INFO, a Linux feature
+    #
+    # Discussed in https://github.com/Homebrew/homebrew-core/pull/88595
+    ./socat-fix-feature-check-tcpinfo.patch
```

Comment:
Is this not used in Linux though?

Reply:
The patch just adds an `#ifdef TCP_INFO`, which won't actually change anything on Linux.

Reply:
Before introducing the patch, the `TCP_INFO` would've had to be defined in order to properly compile. So since the Linux build did compile without the patch, there shouldn't be any loss of functionality with it.